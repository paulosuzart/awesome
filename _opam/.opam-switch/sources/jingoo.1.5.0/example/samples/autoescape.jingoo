{%- set script = "<script></script>" -%}
{%- autoescape true -%}
{{ script }}
{% endautoescape -%}
{%- autoescape false -%}
{{ script }}
{% endautoescape -%}
{%- autoescape true -%}
{{ script | safe + '' | safe }}
{% endautoescape -%}
