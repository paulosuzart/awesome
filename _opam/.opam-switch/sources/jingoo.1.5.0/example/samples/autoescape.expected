&lt;script&gt;&lt;/script&gt;
<script></script>
<script></script>
